utilities/tools/Set-ModuleReadMe.ps1

```diff
@@ -497,8 +497,8 @@ function Set-DeploymentExamplesSection {
                         $requiredParameterEndIndex = $requiredParameterStartIndex
                     }
 
-                    # Add a comment where the non-required parameters start
-                    $bicepExampleArray = $bicepExampleArray[0..$requiredParameterEndIndex] + ('{0}// Non-required parameters' -f (' ' * $requiredParameterIndent)) + $bicepExampleArray[(($requiredParameterEndIndex + 1) .. ($bicepExampleArray.Count))]
+                    # Add a comment where the additional parameters start
+                    $bicepExampleArray = $bicepExampleArray[0..$requiredParameterEndIndex] + ('{0}// Additional parameters' -f (' ' * $requiredParameterIndent)) + $bicepExampleArray[(($requiredParameterEndIndex + 1) .. ($bicepExampleArray.Count))]
```

Comment:
Requires the script to be rerun on all modules to change `Non-required` to `Additional` for all readmes

Reply:
Personally I think 'non-required' is more descriptive than 'additional'

Reply:
Then why committing the suggestion in the first place 😝 The current code would update readmes to `Additional`

Reply:
Let's discuss this for a subsequent PR